src/test/java/edu/cornell/kfs/rass/batch/service/impl/RassServiceImplTest.java

```diff
@@ -96,6 +97,7 @@ public void setUp() throws Exception {
         
         rassUpdateService = springContext.getBean(RassTestConstants.RASS_UPDATE_SERVICE_BEAN_NAME, TestRassUpdateServiceImpl.class);
         rassUpdateService.setDocumentTracker(this::processMaintenanceDocument);
+        rassUpdateService.setDocumentProcessingQueue(Mockito.mock(DocumentProcessingQueue.class));
```

Comment:
If you end up making the service-injecting change noted earlier, you may have to change this unit test setup as well. You will likely have to update "cu-spring-rass-service-test.xml" and the RassMockServiceFactory class so that you can inject a mock DocumentProcessingQueue instance.